### PR TITLE
Updated Starz Presence

### LINKED ISSUE #1490: Starz Rich Presence Bug (Not showing movie or show activity)
**Describe the Bug**

I've been watching a movie in STARZ for about 45 mins now and the RP still shows "STARZ - Browsing" it doesn't show the movie name & time elapsed. I've refreshed the site, disabled/enabled the extension, and none of that seems to fix the issue
**Steps to Reproduce**

1. Log in to STARZ
2. Choose Movie
3. Play Movie
3. Go to discord - it will say "STARZ - Browsing" 

**Expected Behavior**

I expected the rich presence to show the movie name & time elapsed of the movie I'm currently watching
**Screenshots**

https://i.imgur.com/gtONCYz.png - 45 mins in 
https://i.imgur.com/duoh4DG.png - still shows browsing
**Environment**

- OS:  Windows 10
- Browser:  Chrome
- Version:  Version 81.0.4044.129 (Official Build) (64-bit)
  

**Additional context**

I just tried to see if it works for shows, and it's doing the same thing as well. Now it's not even showing the RP on my profile, I keep refreshing & it comes and goes, definitely something wrong behind the scenes

Here's what it shows in console - https://i.imgur.com/8HlLGdR.png

### PR DESCRIPTION
**Pull request by kagbogre**
Fixed movie/series info (also removed subtitle since it’s no longer available in the HTML markup)
Fixed live TV info
Added routing for multiple URLs with elapsed time
Added Contributor

Before:
image

After:
image

closes #1490

### CODE REVIEW

File: websites/S/Starz/dist/metadata.json [hunk 1]
```diff
@@ -3,12 +3,18 @@
     "name": "Bryan",
     "id": "293775922757238784"
   },
+  "contributors": [
+    {
+      "name": "Kedi",
+      "id": "238668511222693888"
+    }
+  ],
```
Comment (Bas950) [inline websites/S/Starz/dist/metadata.json:11]: If you want to you can make yourself author and bryan contributor. Since Bryan doesn't help out anymore 😢

**Review by Bas950 — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issue.

File: websites/S/Starz/dist/metadata.json (hunk 1 above)
Comment (kagbogre) [reply websites/S/Starz/dist/metadata.json]: Rest in peace Bryan

**Review by Bas950 — DISMISSED** [review]

File: websites/S/Starz/dist/metadata.json [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
   "service": "Starz",
   "description": {
     "en": "Starz is an American premium cable and satellite television network which serves as the flagship service of Starz Inc., which is ultimately owned by Lionsgate. Starz's programming features theatrically released motion pictures and first-run original television series."
   },
   "url": "www.starz.com",
-  "version": "1.1.6",
+  "version": "1.2.0",
   "logo": "https://i.gyazo.com/c5acd3cb0ba6e70d4a3ba9f84f8822a1.png",
   "thumbnail": "https://i.gyazo.com/5a52a7bd1f996dbc6876d9d32f8133d5.png",
```
Comment (ririxichan) [inline websites/S/Starz/dist/metadata.json:19]: Upload to imgur and use i.imgur links.

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/S/Starz/dist/metadata.json (hunk 2 above)
Comment (kagbogre) [reply websites/S/Starz/dist/metadata.json:19]: Oh wow, didn’t notice that. I’ll change it

**Comment by kagbogre** [comment]
Created a new app ID for the presence since Bryan (previous author) doesn’t help out anymore

**Review by ririxichan — APPROVED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Since you have changed app ID you need to upload new proof of it working.

**Comment by kagbogre** [comment]
> Since you have changed app ID you need to upload new proof of it working.

image

**Review by Bas950 — APPROVED** [review]
Since you have changed app ID you need to upload new proof of it working.

**Pull request merged**